src/openms/include/OpenMS/CHEMISTRY/IsotopeDistribution.h

```diff
@@ -224,6 +239,14 @@ namespace OpenMS
     /// convolves the distribution @p input with itself and stores the result in @p result
     void convolveSquare_(ContainerType & result, const ContainerType & input) const;
 
+    /** @brief calculates the fragment distribution for a fragment molecule and stores it in @p result.
+
+        @param fragment_isotope_dist the isotopic distribution of the fragment (as if it was a precursor).
+        @param comp_fragment_isotope_dist the isotopic distribution of the complementary fragment (as if it was a precursor).
+        @param precursor_isotopes a list of which precursor isotopes were isolated
+     */
+    void calcFragmentIsotopeDist_(ContainerType& result, const ContainerType& fragment_isotope_dist, const ContainerType& comp_fragment_isotope_dist, const std::vector<UInt>& precursor_isotopes);
```

Comment:
you are right, there is no policy, it was just my (subjective) general impression and I think it would make it easier to read the API if you can expect certain things to hold (like order of parameters).
But if the old methods in this class do not follow, then it would also make it inconsistent here... so just leave as is.